pkgs/applications/video/celluloid/default.nix

```diff
@@ -46,8 +46,6 @@ stdenv.mkDerivation rec {
 
   postPatch = ''
     patchShebangs meson-post-install.py src/generate-authors.py
-    # Remove this for next release
-    substituteInPlace meson-post-install.py --replace "gtk-update-icon-cache" "gtk4-update-icon-cache"
```

Comment:
Included in 0.23 (https://github.com/celluloid-player/celluloid/commit/bbb4d4f06a5a0e4b9a43b9af2e94b5fa0801d395)